### PR TITLE
io.github.qwersyk.Newelle

### PR DESCRIPTION
**Pull request by qwersyk**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.github.qwersyk.Newelle.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "newelle",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/qwersyk/Newelle.git"
```
Comment (hfiguiere) [inline io.github.qwersyk.Newelle.json:35]: you need  a specific commit or tag here.

File: io.github.qwersyk.Newelle.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "newelle",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak",
```
Comment (hfiguiere) [inline io.github.qwersyk.Newelle.json:13]: why do you need this dbus permission?

**Comment by qwersyk** [comment]
Adding tag and commit.

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
I would like to build my program with "--talk-name=org.freedesktop.Flatpak" permission.

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by hfiguiere** [comment]
> I would like to build my program with "--talk-name=org.freedesktop.Flatpak" permission.

why do you need it. You never answered the question.

**Comment by qwersyk** [comment]
> > I would like to build my program with "--talk-name=org.freedesktop.Flatpak" permission.
> 
> why do you need it. You never answered the question.

I need this so that the program can do commands outside the box so that my program can do more things, when my program is running in the box it has many limitations and can do little. The idea of my program is that the user can ask the program to do something and the program will do it, crop the video, change the encoding of the image and so on. is only limited by what programs are installed, but when my program is in the box, when you try to run, for example, the command: "convert -size 100x100 xc:white image.png", you get an error: "convert: command not found", so subprocess.Popen("flatpak-spawn --host convert..."), but without "--talk-name=org.freedesktop.Flatpak", you get an error: "org.freedesktop.DBus.Error.ServiceUnknown".

**Comment by hfiguiere** [comment]
if you want ImageMagick, build it in the flatpak. ImageMagick is not default on many systems anyway.

**Comment by qwersyk** [comment]
> if you want ImageMagick, build it in the flatpak. ImageMagick is not default on many systems anyway.

No, that was just an example, my program is like a terminal, only you write in words and the neural network converts that into commands that are run.

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
bot, build io.github.qwersyk.Newelle

**Comment by qwersyk** [comment]
Thank you all so much for your help!

**Pull request closed without merging**